projects/storefrontapp/src/assets/i18n/payment-review.en.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
As discussed offline, this file should be rather something like `order-summary` or `order-details`. 
And current `order-summary` should become something like `order-cost` or `order-value`

Reply:
why not order-review?